Sources/Conformance/main.swift

```diff
@@ -24,7 +24,7 @@ import Darwin.C
 import Foundation
```

Comment:
Is Foundation still needed for something else?

Reply:
Yes, there is the `func writeResponse(data: Data)` method. This ends up taking the result of calling `serializedData()` on a proto, so it will be changed once the serialisation path is rid of `Data`.